Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -1436,38 +1436,15 @@ static void paintAttachmentIconBackground(const RenderAttachment& attachment, Gr
     }
 }
 
-static bool shouldDrawIcon(const String& title)
-{
-#if HAVE(QUICKLOOK_THUMBNAILING)
```

Comment:
I see you're removing all the `QUICKLOOK_THUMBNAILING` uses.
I think you missed the ones in Tools/TestWebKitAPI/Tests/WebKitCocoa/WKAttachmentTests.mm .
You may want to do a find-all, in case there are more stragglers.

Reply:
Good call!